mona_sdk/client.py

```diff
@@ -98,16 +98,33 @@ class Client:
     API.
     """
 
-    def __init__(self, api_key, secret):
+    def __init__(
+        self,
+        api_key,
+        secret,
+        raise_authentication_exceptions=None,
```

Comment:
same comment regarding the default from the env var

Reply:
Ok got it, I moved all relevant env vars creations to client.py and set them to be the default values for the clients fields in case they are not supplied and then only the fields will be in use. lmk if it looks fine now:)
@nemo-mona